### ISSUE TITLE
Clarify "excludes"

### ISSUE THREAD

**Issue by kwin** [mojohaus/license-maven-plugin#366]
The documentation at https://www.mojohaus.org/license-maven-plugin/check-file-header-mojo.html#excludes does not really state:

- Against which path the exclude is compared (full absolute file name, module relative file name, file title)
- Which wildcard patterns are supported?

**Comment by kwin** [comment]
I just figured out by looking at looking at https://github.com/mojohaus/license-maven-plugin/blob/01fd9a1c560fb608e721de0cb1673a29805a40cd/src/main/java/org/codehaus/mojo/license/AbstractLicenseNameMojo.java#L291 that excludes are matched against the relative path below the roots (https://www.mojohaus.org/license-maven-plugin/check-file-header-mojo.html#roots).

**Comment by vatbub** [comment]
I agree with that: I had to look up `DirectoryScanner` docs from plexus utils to understand the available wildcard patterns
